cukinia

```diff
@@ -621,6 +622,37 @@ _cukinia_mount()
 	return 0
 }
 
+# _cukinia_i2c: Check if i2c device exists
+# Optional: Check of a device address
+# arg1: the bus number
+# arg2: (optional) the device address
+_cukinia_i2c()
+{
+	local bus_number="$1"
+	local device_address="$2"
+	local ret
+
+	_cukinia_prepare "Checking if i2c-$bus_number exists"
+
+	test -d /sys/bus/i2c/devices/i2c-$bus_number
+	ret=$?
```

Comment:
rather than test -d, ret=.., if $ret , use if directly . `if [ ! -d /path ]; then`

also, keep the coding style uniform with the rest of the file (if..then on the same line)